### PR TITLE
Power tags are displayed as mini cards in the sidebar

### PR DESCRIPTION
**Pull request by Manasa2850**
Part of #9668

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by Manasa2850** [comment]
Screenshot 2021-06-24 00:24:31

**Comment by Manasa2850** [comment]
@jywarren @RuthNjeri could you please help me out with this error `Capybara::Ambiguous: Ambiguous match, found 3 elements matching visible css ".tags-list .card-body .ellipsis"`. I need to delete 1 of the 3 tags but they're all identical when it comes to css. How do I select one particular tag?
Thanks!

File: test/system/post_test.rb [hunk 1]
```diff
@@ -220,8 +222,8 @@ def assert_page_does_not_reload(message = "page should not reload")
     find('a#tags-open').click()
 
     # Make sure proper latitude and longitude tags are added
-    assert_selector('.tags-list .badge a[href="/tag/lat:22"]', text: "lat:22")
-    assert_selector('.tags-list .badge a[href="/tag/lon:76"]', text: "lon:76")
+    assert_selector('.tags-list .card-body a[href="/tag/lat:22"]', text: "lat:22")
+    assert_selector('.tags-list .card-body a[href="/tag/lon:76"]', text: "lon:76")
```
Comment (TildaDares) [inline test/system/post_test.rb:226]: @Manasa2850 Have you tried adding `count: 1`?

File: test/system/post_test.rb [hunk 2]
```diff
@@ -38,26 +38,28 @@ def setup
...
 
     find('a#tags-open').click()
 
-    # There should be 1 tag that shows up as a badge and 2 as a card
-    page.assert_selector('.tags-list .card-body', :count => 2)
-    page.assert_selector('.tags-list p.badge', :count => 1)
+    # There should be 3 tags that show up as a card
+    page.assert_selector('.tags-list .card-body', :count => 3)
+
+    find('.tags-list:first-of-type .card-body .ellipsis').click()
```
Comment (Manasa2850) [inline test/system/post_test.rb:55]: @TildaDares this is the line which causes the system test to fail. I used `first-of-type` to select only the first tag but I'm not sure why it still fails.

Comment (TildaDares) [reply test/system/post_test.rb:55]: @Manasa2850 Have you tried using `first`? https://www.rubydoc.info/gems/capybara/Capybara%2FNode%2FFinders:first

Comment (Manasa2850) [reply test/system/post_test.rb]: Thank you so much @TildaDares!! That worked :tada:

**Review by RuthNjeri — APPROVED** [review]
This looks good to me @Manasa2850 🎉

**Comment by jywarren** [comment]
Hi all, this looks great! I see your follow-up in #9849 and wanted to note that you can open a PR against another branch, i.e. against another PR - so that this and the other could be merged together. Or you could do it so that that one includes the commits from this one. Either way, right now we have more than 2 showing on the stable server, which is not a critical problem but we would generally like to avoid having stable in a state where it's not publishable to the live site. I hope this makes sense, and great work overall!!

image

https://stable.publiclab.org/wiki/coqui

**Pull request merged**